### PR TITLE
Feature/templates

### PR DESCRIPTION
**Pull request by ridlees**
Adding custom Github templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_practice.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Practice suggestion
+about: Suggest new article about a good Practice 
+title: "[new_practice]"
+labels: 'article_type: practice'
```
Comment (prokopsimek) [inline .github/ISSUE_TEMPLATE/new_practice.md:5]: maybe to add label "suggestion"?

File: .github/ISSUE_TEMPLATE/new_practice.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+name: Practice suggestion
+about: Suggest new article about a good Practice 
+title: "[new_practice]"
```
Comment (prokopsimek) [inline .github/ISSUE_TEMPLATE/new_practice.md:4]:
```suggestion
title: "New article: "
```

File: .github/ISSUE_TEMPLATE/new_problem.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+name: Problem suggestion
+about: Suggest new article about new problem
+title: "[new_problem]"
```
Comment (prokopsimek) [inline .github/ISSUE_TEMPLATE/new_problem.md:4]:
```suggestion
title: "New article: "
```

File: .github/ISSUE_TEMPLATE/new_problem.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+labels: 'article_type: problem'
```
Comment (prokopsimek) [inline .github/ISSUE_TEMPLATE/new_problem.md:5]: add "suggestion" label?

File: .github/ISSUE_TEMPLATE/new_problem.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+assignees: ''
+
+---
+# Title: [Article] %Problem name
+
+What is this about: “What is this problem about”? Which existing (or new) practices can solve this problem? 
+
+Read more here: The starting point for your research (You can use different sources if it is more fitting)
+
```
Comment (prokopsimek) [inline .github/ISSUE_TEMPLATE/new_problem.md:14]: what about to add checklist:
```
- [ ] the practice does not exist yet

## Are there any similar practices? What differs?
...
```

File: .github/ISSUE_TEMPLATE/new_practice.md (hunk 1 above)
Comment (ridlees) [reply .github/ISSUE_TEMPLATE/new_practice.md:5]: What about the existing label enhancement?

File: .github/ISSUE_TEMPLATE/new_problem.md (hunk 5 above)
Comment (ridlees) [reply .github/ISSUE_TEMPLATE/new_problem.md:14]: added in the new commit

**Review by prokopsimek — APPROVED** [review]

**Pull request merged**